options.ml

```diff
@@ -31,6 +31,12 @@ module Run = False
     let help = "generate a random C function"
   end)
 
+module CheckAst = False
```

Comment:
This can be `True` by default. I benchmarked it with `time (for i in `seq 1 1000`; do frama-c -ldrgen -ldrgen-seed $i -ldrgen-check-ast > /dev/null; done)`, and the runtime cost seems to be on the order of 1-2%, which I'm happy to pay.

Reply:
OK, that's fine with me.